### PR TITLE
Custom validation message for Crowdfund form primary currency

### PR DESCRIPTION
**Pull request by BenSanex**
Second half of #589

### CODE REVIEW

File: BTCPayServer/Models/AppViewModels/UpdateCrowdfundViewModel.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ public class UpdateCrowdfundViewModel
         public DateTime? StartDate { get; set; }
         public DateTime? EndDate { get; set; }
 
-        [Required]
+        [Required(ErrorMessage="The primary currency is required")]
         [MaxLength(5)]
         [Display(Name = "The primary currency used for targets and stats. (e.g. BTC, LTC, USD, etc.)")]
```
Comment (britttttk) [inline BTCPayServer/Models/AppViewModels/UpdateCrowdfundViewModel.cs]: It follows the style of the other missing field messages on this page, if you just remove `The` from the Display Name. Also need asterisk here :)

Comment (BenSanex) [reply BTCPayServer/Models/AppViewModels/UpdateCrowdfundViewModel.cs]: The asterisk should get taken care of when rockstars PR gets merged, didn't want to accidentally cause a merge conflict.
IMO repeating `used for targets and stats. (e.g. BTC, LTC, USD, etc.)` is redundant but can change it if that's preferred.

Comment (britttttk) [reply BTCPayServer/Models/AppViewModels/UpdateCrowdfundViewModel.cs]: There are three required fields: Title, Description, Primary Currency. The title field asterisk already exists. Rockstardev has added one for the description field, so there is one missing for the third field. (:

Comment (BenSanex) [reply BTCPayServer/Models/AppViewModels/UpdateCrowdfundViewModel.cs]: Alright I got it this time :D

**Comment by rockstardev** [comment]
Whoho

**Pull request merged**